src/main/java/com/refinedmods/refinedstorage/item/group/MainItemGroup.java

```diff
@@ -12,6 +13,6 @@ public MainItemGroup() {
 
     @Override
     public ItemStack createIcon() {
-        return new ItemStack(RSBlocks.CREATIVE_CONTROLLER.get());
+        return new ItemStack(RSBlocks.CREATIVE_CONTROLLER.get(DyeColor.LIGHT_BLUE).get());
```

Comment:
We should probably move `DyeColor.LIGHT_BLUE` to a constant in `BlockUtils.DEFAULT_COLOR`.